data_steward/gcloud/bq/__init__.py

```diff
@@ -227,20 +229,31 @@ def define_dataset(self, dataset_id: str, description: str,
 
         return dataset
 
-    def copy_dataset(self, input_dataset: str, output_dataset: str) -> list:
+    def copy_dataset(self,
+                     input_dataset: str,
+                     output_dataset: str,
+                     job_config: CopyJobConfig = None) -> list:
         """
         Copies tables from source dataset to a destination datasets
 
         :param input_dataset: fully qualified name of the input(source) dataset
         :param output_dataset: fully qualified name of the output(destination) dataset
+        :param job_config: An optional google.cloud.bigquery.job.CopyJobConfig
         :return: incomplete jobs
         """
+        job_config = job_config if job_config else CopyJobConfig(
+            write_disposition=WriteDisposition.WRITE_EMPTY)
         # Copy input dataset tables to backup and staging datasets
         tables = super(BigQueryClient, self).list_tables(input_dataset)
         job_list = []
         for table in tables:
             staging_table = f'{output_dataset}.{table.table_id}'
-            job = self.copy_table(table, staging_table)
+            job_config.labels.update({
+                'table_name': table.table_id,
+                'copy_from': input_dataset,
+                'copy_to': output_dataset
```

Comment:
@MichaelSchmidt82 I just realized, since you are leaving these here you must ensure the label values are always [lower case](https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements).  You should wrap these as:
```suggestion
                'table_name': table.table_id.lower(),
                'copy_from': input_dataset.lower(),
                'copy_to': output_dataset.lower()
```